test cases/common/36 tryrun/meson.build

```diff
@@ -14,10 +14,7 @@ endif
 ok_code = '''#include<stdio.h>
 int main(void) {
   printf("%s\n", "stdout");
-  fprintf(stderr, "%s\n", "stderr");
-  return 0;
-}
-'''
+''' + 'fprintf(stderr, "%s\\n", "stderr"); return 0; }'
```

Comment:
I don't understand why this change is necessary.

Reply:
The current description is:
> _Cosmetic_ fixes submitted without the major code change in PR #6241 just to make sure they pass CI.

Reply:
Cosmetically, it looks worse, so that doesn't explain why.

Reply:
Should I have copied the commit message in the description? That seems to be common practice on github which treats commit messages poorly.